src/app/component/CategoryProductList/CategoryProductList.style.scss

```diff
@@ -42,15 +41,28 @@
         animation: appear 100ms 200ms ease-in forwards;
     }
 
+    &-More {
+        min-height: 3rem;
+    }
+
+    &-LoadButton {
+        background-color: var(--color-load-button-background);
+        cursor: pointer;
+        font-size: 13pt;
+        margin: 1em 0;
+        padding: 1em 0;
+        text-align: center;
+
+        &_hidden {
+            display: none;
```

Comment:
You want this button be inaccessible to crawlers as well? I am not sure. But this property makes it disappear from it's vision.

Reply:
Moved this bit to pagination, now it is so damn good :)